app/src/main/java/com/example/android/architecture/blueprints/todoapp/data/source/DefaultTasksRepository.kt

```diff
@@ -95,68 +83,51 @@ class DefaultTasksRepository(
         }
     }
 
-    override fun getTaskStream(taskId: String): Flow<Task?> {
-        return tasksDao.observeTaskById(taskId).map { it.toExternalModel() }
+    private suspend fun sendTasksToNetworkDataSource() {
```

Comment:
`sendTasksToNetworkDataSource` sounds too prescriptive with what the network is doing. Maybe we should choose a more generic name? We shouldn't know the network data source impl details.

Maybe something something shorter and more generic like `syncNetworkTasks` would do

Reply:
I agree this name is too verbose, however, the direction of data travel and where it's going to is important. For this reason `sync` is too generic. How about just `sendTasksToNetwork`?

Reply:
btw I chose this name to match the existing counterpart method `updateTasksFromNetworkDataSource`, I could also rename this to `loadTasksFromNetwork`.